Src/WitsmlExplorer.Api/Routes.cs

```diff
@@ -1,55 +1,63 @@
+using System.Collections.Generic;
+
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Http;
 
 using WitsmlExplorer.Api.HttpHandler;
+using WitsmlExplorer.Api.Models;
 
 namespace WitsmlExplorer.Api;
 
 public static class Api
 {
     public static void ConfigureApi(this WebApplication app)
     {
-        app.MapGet("/api/witsml-servers", WitsmlServerHandler.GetWitsmlServers);
-        app.MapPost("/api/witsml-servers", WitsmlServerHandler.CreateWitsmlServer);
-        app.MapMethods("/api/witsml-servers/{witsmlServerId}", new[] { HttpMethods.Patch }, WitsmlServerHandler.UpdateWitsmlServer);
-        app.MapDelete("/api/witsml-servers/{witsmlServerId}", WitsmlServerHandler.DeleteWitsmlServer);
+        app.MapGet("/api/witsml-servers", WitsmlServerHandler.GetWitsmlServers).Produces<IEnumerable<Server>>().Produces(StatusCodes.Status500InternalServerError);
```

Comment:
I propose that you add some linebreaks to improve the readability a bit here:
```
app.MapGet("/api/witsml-servers", WitsmlServerHandler.GetWitsmlServers)
   .Produces<IEnumerable<Server>>()
   .Produces(StatusCodes.Status500InternalServerError);
```

You could also take a look at `builder.Services.AddEndpointsApiExplorer()`, which should annotate endpoints automatically without these `.Produces` statements (although I suspect it might not correctly annotate error codes)

Reply:
Seems like we were already using `AddEndspointsApiExplorer` and it might also be required 🤔 

It also does not include the model description for the endpoint results, so there are trade-offs that can be made here. I like that the OpenAPI UI describes the result models, but I'm not a huge fan of adding all these `.Produces` calls 🤷

Reply:
Another alternative to adding all these `.Produces` calls in the route configurations is to use the Produces attribute on the handler methods as well:
```
[Produces(typeof(IEnumerable<Server>))]
```
This is closer to the actual implementation

Reply:
Yes that is a good idea, it is close to the handler, so that makes sense.
We usually only return 200 statuscode and 500. I guess they can be attributed as well 👍